### PR TITLE
Fix pymedusa/Medusa/issues/1372

### PR DESCRIPTION
**Pull request by fernandog**
Fixes https://github.com/pymedusa/Medusa/issues/1372
torrent_rows  is a list of dict

@medariox @duramato @FireBladeNooT

thanks to pycharm

image

### CODE REVIEW

**Comment by fernandog** [comment]
notice that rss search is not ordered by date:

image

it may impact in the daily search "avoid reprocess same item" feature @p0psicles right?

**Comment by fernandog** [comment]
any idea why this?

```
  File "/home/osmc/Medusa/medusa/providers/torrent/json/t411.py", line 102, in 
    sorted_jdata = sorted(jdata, key=lambda k: k[b'added'], reverse=True)
TypeError: string indices must be integers
```

**Comment by fernandog** [comment]
@medariox review/approve please?

File: medusa/providers/torrent/json/t411.py [hunk 1]
```diff
@@ -108,25 +108,28 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
...
         :param mode: The current mode used to search, e.g. RSS
 
         :return: A list of items found
         """
-
         items = []
 
-        torrent_rows = data.get('torrents') if mode != 'RSS' else data
+        unsorted_torrent_rows = data.get('torrents') if mode != 'RSS' else data
+        torrent_rows = sorted(unsorted_torrent_rows, key=itemgetter(u'added'), reverse=True)
```
Comment (medariox) [inline medusa/providers/torrent/json/t411.py]: why the u''? we use unicode_literals

Comment (fernandog) [inline medusa/providers/torrent/json/t411.py]: done

**Pull request merged**